### DISCUSSION TITLE
2.5” drives corrupts plots

### DISCUSSION THREAD

**Discussion by elmz3000** [Chia-Network/chia-blockchain#6873] [Support]
I bought several 2.5” drives to extend my Chia Rig , I can successfully plot from an SSD to my 3.5” HDD but when I then try to move from the 3.5” to the 2.5” drives the plots corrupt. Some actually not instantly. In Ubuntu the healthy plots have a 1&0s icon where the corrupted plots then have a text file icon. Everything works totally fine from SSD to 3.5 but as soon as I move them they corrupt either instantly or after a few hours. A rare few don’t corrupt at all. It’s very confusing and inconsistent. I also tried different file systems already. exFat, F2FS and HFS same for all of them. They are new 4TB drives non mechanical flash memory , fairly budget. Slow transfer speeds.

Any help would be highly appreciated.
Cheers,
Elmar

**Comment by BrandtH22** [comment]
@elmz3000 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
